File: src/NubipApp.Models/Request.cs
using System;

namespace NubipApp.Models
{
    public class Request
    {
        public int RequestId { get; set; }
        public int LessonId { get; set; }
        public int TimeId { get; set; }
        public DateTime From { get; set; }
        public DateTime To { get; set; }
        public DateTime RequestTime { get; set; }
        public bool Status { get; set; }

        public Lesson Lesson { get; set; }
        public Time Time { get; set; }
    }
}

File: src/models/request.ts
import mongoose from "mongoose";

let Schema = mongoose.Schema;

let RequestSchema = new Schema({
	requestId: Number,
	lessonId: [{
		type: Schema.Types.ObjectId, ref: 'Lesson'
	}],
	timeId: [{
		type: Schema.Types.ObjectId, ref: 'Time'
	}],
	from: Date,
	to: Date,
	requestTime: Date,
	status: Boolean
});

let Request = mongoose.model('Request', RequestSchema);

export default Request;